### PR TITLE
Upgrade Gradle to 2.14

### PR DESCRIPTION
**Pull request by ljank**
While upgrading to Gradle 2.14 I've found that cucumber plugin stopped working due to moved classes. Decided to act just like these guys: https://github.com/x2on/gradle-hockeyapp-plugin/pull/86

@saulius @vidma @astrauka

### CODE REVIEW

**Comment by astrauka** [comment]
:+1:

File: build.gradle [hunk 1]
```diff
@@ -27,11 +27,11 @@ dependencies {
...
 
-    //testCompile 'org.spockframework:spock-core:0.7-groovy-2.0'
-    testCompile 'com.netflix.nebula:nebula-test:3.1.0'
+    //testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    testCompile 'com.netflix.nebula:nebula-test:4.2.2'
 
-    //intTestCompile 'org.spockframework:spock-core:0.7-groovy-2.0'
-    integTestCompile 'com.netflix.nebula:nebula-test:3.1.0'
+    //intTestCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    integTestCompile 'com.netflix.nebula:nebula-test:4.2.2'
```
Comment (ljank) [inline build.gradle:34]: This dude uses Spock framework that was incompatible with upgraded Groovy version (2.3 -> 2.4), had to update this dependency too. `./gradlew test` is successful, assuming everything's OK here :)

File: gradlew.bat [hunk 2]
```diff
@@ -46,7 +46,7 @@ echo location of your Java installation.
 goto fail
 
 :init
-@rem Get command-line arguments, handling Windowz variants
+@rem Get command-line arguments, handling Windows variants
```
Comment (ljank) [inline gradlew.bat:49]: I like this change the most :)

**Comment by saulius** [comment]
:+1:

**Pull request merged**